lib/libtlspool.c

```diff
@@ -37,6 +38,18 @@ static pthread_cond_t updated_poolfd = PTHREAD_COND_INITIALIZER;
 
 static pthread_mutex_t prng_lock = PTHREAD_MUTEX_INITIALIZER;
 
+int tlspool_getpid () {
+	int fd;
+
+	if ((fd = open("/var/run/tlspool.pid", O_RDONLY)) != -1) {
+		char str_pid[256];
+
+		if (read(fd, str_pid, sizeof(str_pid)) != -1) {
```

Comment:
Can you finish the function tlspool_getpid(void) function so it's cryptographically safe and doesn't contain hardcoded paths?